gulp-tasks/build.js

```diff
@@ -23,7 +23,7 @@ const fsePromise = promisify('fs-extra');
 
 const printHeading = (heading) => {
   /* eslint-disable no-console */
-  process.stdout.write(chalk.inverse(`  ⚒️  ${heading}  `));
+  process.stdout.write(chalk.inverse(`  ⚒  ${heading}  `));
```

Comment:
There was some weird control character in this line that would result in a primitive smiley face being printed out to the console on my Mac.